models/common.py

```diff
@@ -296,7 +296,7 @@ def __init__(self, weights='yolov5s.pt', device=torch.device('cpu'), dnn=False,
         super().__init__()
         w = str(weights[0] if isinstance(weights, list) else weights)
         pt, jit, onnx, xml, engine, coreml, saved_model, pb, tflite, edgetpu, tfjs = self.model_type(w)  # get backend
-        stride, names = 64, [f'class{i}' for i in range(1000)]  # assign defaults
+        stride, names = 32, [f'class{i}' for i in range(1000)]  # assign defaults
```

Comment:
Seems the P6 series need stride 64, so it would be more helper to set this parameter optional here.

Reply:
sorry, i can't realize the meaning of P6, do you mean as follow? i think the data configuration is used to config something about dataset, it's not suitable to set stride on it?
![image](https://user-images.githubusercontent.com/50093021/162352893-ceea9abf-d846-4c44-bd70-627372392ace.png)

Reply:
sorry，i know， set it on cmd argument， i try it

Reply:
P6 is yolov5n6, yolov5s6, yolov5m6 and yolov5l6, introduced from tag 5.0.

Reply:
thanks！and i have added argument --stride